src/driver/sqljs/SqljsDriver.ts

```diff
@@ -209,7 +209,14 @@ export class SqljsDriver extends AbstractSqliteDriver {
             this.databaseConnection = new this.sqlite.Database();
         }
 
-        return Promise.resolve(this.databaseConnection);
+        return new Promise<void>(async (ok, fail) => {
```

Comment:
`async` shouldn't be needed here, right?

Reply:
Indeed